app/forms/work_packages/types/subject_configuration_form.rb

```diff
@@ -82,6 +82,20 @@ def enterprise?
       def validation_message_for(attribute)
         model.validation_errors.messages_for(attribute).to_sentence.presence
       end
+
+      def pattern_input_caption
+        I18n.t(
+          "types.edit.subject_configuration.pattern.caption_with_supported_attributes_link",
+          link: make_link(
+            ::OpenProject::Static::Links.url_for(:enterprise_features, :work_package_subject_generation),
```

Comment:
We need to double check the correctness of this link.

I _heard_ that the docs for this are already prepared, but on the other hand there is nothing but emptyness under this already existing link destination.